src/main/webapp/app/exercises/shared/team/teams-import-dialog/teams-import-from-file-form.component.ts

```diff
@@ -73,11 +104,58 @@ export class TeamsImportFromFileFormComponent {
         this.changeDetector.detectChanges();
     }
 
+    /**
+     * Parses a csv file and returns a promise with a list of rows
+     * @param csv File content that should be parsed
+     */
+    private parseCSVFile(csv: string): Promise<CsvEntry[]> {
+        return new Promise((resolve, reject) => {
+            parse(csv, {
+                download: false,
+                header: true,
+                transformHeader: (header: string) => header.toLowerCase().replace(' ', '').replace('_', ''),
+                skipEmptyLines: true,
+                complete: (results) => resolve(results.data as CsvEntry[]),
+                error: (error: any) => reject(error),
+            });
+        });
+    }
+
+    /**
+     * Parse the content of a csv file to students with teams
+     * @param entries All entries of the csv file
+     */
+    convertCsvEntries(entries: CsvEntry[]): StudentWithTeam[] {
+        return entries.map(
+            (entry) =>
+                ({
+                    registrationNumber: entry[csvColumns.registrationNumber] || entry[csvColumns.matrikelNummer] || entry[csvColumns.matriculationNumber] || undefined,
+                    username:
+                        entry[csvColumns.login] ||
+                        entry[csvColumns.username] ||
+                        entry[csvColumns.user] ||
+                        entry[csvColumns.benutzer] ||
+                        entry[csvColumns.benutzerName] ||
+                        undefined,
+                    firstName: entry[csvColumns.firstName] || entry[csvColumns.vorname] || undefined,
+                    lastName:
+                        entry[csvColumns.lastName] ||
+                        entry[csvColumns.familyName] ||
+                        entry[csvColumns.surname] ||
+                        entry[csvColumns.name] ||
+                        entry[csvColumns.nachname] ||
+                        undefined,
+                    teamName: entry[csvColumns.teamName] || entry[csvColumns.team] || entry[csvColumns.gruppe] || undefined,
+                } as StudentWithTeam),
+        );
+    }
+
     /**
      * Convert imported team list to normal teams
      */
     convertTeams(importTeam: StudentWithTeam[]): Team[] {
         const teams: Team[] = [];
+        let entryNr = 1;
         importTeam.forEach((student) => {
```

Comment:
```suggestion
        importTeam.forEach((student, index) => {
```
You can the remove `entryNr` and use `index + 1`